### DISCUSSION TITLE
Dependency issue - anvil_extras

### DISCUSSION THREAD

**Discussion by psychedel** [anvilistas/amoni#9] [General]
Thank you for the awesome tool @owen.campbell !
May I ask - how to add anvil_extras as working dependency?

I receive error
```

The app you requested could not be loaded.

This app may be misconfigured. The following error occurred:

App dependency not found

```

But I did everything from manual and anvil_extras is the single dependency of my app..
I've completed this step -
```
amoni app add --as-dependency https://github.com/anvilistas/anvil-extras.git anvil_extras C6ZZPAPN4YYF5NVJ
```

I see in my config.yaml that
```
dep_id:
  C6ZZPAPN4YYF5NVJ: anvil_extras
```
And I have anvil_extras in my app folder.
And in anvil.yaml of my Anvil application I see the same dependency id
```
- app_id: C6ZZPAPN4YYF5NVJ
  version: {dev: false}
```

**Comment by meatballs** [comment]
Can you share a link to the repo for your project?

**Reply by psychedel** [reply]
Yes, thank you - here is it
https://github.com/psychedel/amoni_env

**Comment by meatballs** [comment]
See #10 

Should now be fixed in v0.0.9

**Comment by meatballs** [comment]
You'll need to make a manual change in your existing repo but the new version should stop it happening again.

In `config.yaml`, change `dep_id` to `dep-id`
